### PR TITLE
Certiticates > Add more info for static and manual certificates in UI

### PR DESCRIPTION
**Pull request by pv3nturi**
PR for #196

Changelog:

Add additional information in UI for static/manual certificates:
- status (available/expired)
- expiringDate
- serialNumber

### CODE REVIEW

**Review by hamadodene — APPROVED** [review]
LGTM

**Comment by pv3nturi** [comment]
I gonna add these improvements https://github.com/diennea/carapaceproxy/issues/193 here

File: carapace-server/src/test/java/org/carapaceproxy/api/StartAPIServerTest.java [hunk 1]
```diff
@@ -46,16 +46,20 @@
...
 import org.junit.Test;
 import static org.carapaceproxy.utils.APIUtils.certificateStateToString;
 import static org.carapaceproxy.utils.CertificatesTestUtils.generateSampleChain;
+import static org.carapaceproxy.utils.CertificatesUtils.KEYSTORE_PW;
 import static org.carapaceproxy.utils.CertificatesUtils.createKeystore;
+import static org.hamcrest.MatcherAssert.assertThat;
 import java.security.KeyPair;
 import java.security.cert.Certificate;
 import java.security.cert.X509Certificate;
+import org.apache.curator.shaded.com.google.common.io.Files;
```
Comment (nicoloboschi) [inline carapace-server/src/test/java/org/carapaceproxy/api/StartAPIServerTest.java:60]: Did you mean java.nio.file.Files?

**Review by nicoloboschi — CHANGES_REQUESTED** [review]
Looks good, we need to change an import in a test file IMHO

**Review by nicoloboschi — APPROVED** [review]
+1

**Pull request merged**